e2e/portalicious/pages/UsersPage.ts

```diff
@@ -69,6 +87,42 @@ class UsersPage extends BasePage {
 
     expect(formattedTextContent).toContain(textContent);
   }
+
+  async addNewUser({ fullName, email }: { fullName: string; email: string }) {
+    // Act
+    await this.newUserButton.click();
+    await this.fullNameInput.fill(fullName);
+    await this.emailInput.fill(email);
+    await this.submitButton.click();
+    // Assert
+    await this.validateToastMessage('User added');
```

Comment:
Does it make sense to split the validate steps off in a separate method than the steps actually needed to add the user?

Reply:
Same as below. I'll change it.

Reply:
Done!